pyproject.toml

```diff
@@ -73,6 +73,9 @@ zarr = [
 [tool.ruff]
 line-length = 100
 
+[tool.ruff.lint.flake8-tidy-imports]
+ban-relative-imports = "all"
+
```

Comment:
is this different from adding `TID252` to the list of selected rules below?

Reply:
done

Reply:
This I do not know. I used the documentation to update the pyproject.toml file

https://docs.astral.sh/ruff/settings/#lint_flake8-tidy-imports_ban-relative-imports

Reply:
I would use both, and move this section below the general tool.ruff.lint this way we'll have all specific configuration of ruff.tool.lint grouped together (there are already pydocstyle and file specific configs). From what I understand, this part is changing the default behaviour of the rule which probably adds it to the list of selected rules under the hood (otherwise configuring it would be pointless) but feels a bit obscure. The docs for the rule: https://docs.astral.sh/ruff/rules/relative-imports/. We can add a comment after the rule explaining what it is like for the others.